spinta/backends/mongo.py

```diff
@@ -176,10 +167,7 @@ def getall(context: Context, model: Model, backend: Mongo, **kwargs):
     # Yield all available entries.
     model_collection = backend.db[model.get_type_value()]
     for row in model_collection.find({}):
-        id = str(row.pop('_id'))
-        yield {
-            'id': id,
-        }
+        yield prepare(context, 'getall', model, backend, row)
```

Comment:
I think, here an iterator should be passed to `prepare`, because you need to have control, how all rows are wrapped, for example they can be wrapped into `{"resources": [...]]}` or something like that.

Reply:
Same for postgre.

Reply:
I assumed that exporter would be responsible for this?

https://github.com/atviriduomenys/spinta/blob/master/spinta/config/__init__.py#L73

Reply:
Then, why you need `action` here?

Reply:
Because different actions may have differently structured responses, e.g. POST in spinta only returns the `type` and `id` instead of the whole object.